web/src/components/TimeAverageToggle.tsx

```diff
@@ -17,42 +20,41 @@ export interface TimeAverageToggleProps {
 
 function TimeAverageToggle({ timeAverage, onToggleGroupClick }: TimeAverageToggleProps) {
   const { t } = useTranslation();
-  const options = Object.keys(TimeAverages).map((time) =>
-    createOption(time.toLowerCase() as TimeAverages, t)
+  const options = useMemo(
+    () =>
+      Object.keys(TimeAverages).map((time) =>
+        createOption(time.toLowerCase() as TimeAverages, t)
+      ),
+    [t]
   );
 
   return (
-    <ToggleGroupPrimitive.Root
+    <ToggleGroupRoot
```

Comment:
Would be nice to add a keyboard focus state, seems like it worked better before (not sure why though) :)

Reply:
Good catch, we can go with a simple outline of the pill as an easy fix 🙌 

![image](https://github.com/user-attachments/assets/d1a4f129-1d80-44ad-b1bd-089d9d349229)

Reply:
Are you adding the colors to the Figma or is it the brand color now (but with the dark variant as well)?

Reply:
Brand color, but[ added it to Figma](https://www.figma.com/design/O1DtS6UyZHSH4IymG8Ufsw/%F0%9F%8E%B9-Design-Hack?node-id=5257-5632&t=ntt7JkKDAaO1ylDA-1) as well! 🙌 My bad, I didn't link it before 🙈

Reply:
`shift+tab` two times on a refreshed view takes you to it, then you can go sideways with arrows - so I think it would be nice to just quickly a `focus:outline-brand-green` kinda class as a first step :)

Reply:
Yeah I think that would work!

I'll add it then!